static/js/lib/api_test.js

```diff
@@ -216,6 +218,20 @@ describe('api', function() {
         });
       });
 
+      it('returns expected values when a POST to send a learner email succeeds', () => {
+        fetchJSONStub.returns(Promise.resolve(MAIL_RESPONSE));
+        return sendLearnerMail('subject', 'body', learnerStudentId).then(mailResp => {
+          assert.ok(fetchJSONStub.calledWith(`/api/v0/mail/learner/${learnerStudentId}/`, {
```

Comment:
It would be better to use [`sinon.assert.calledWith()`](http://sinonjs.org/releases/v1.17.7/assertions/#sinonassertcalledwithspy-arg1-arg2-).